packages/shared/src/utils/utils.ts

```diff
@@ -127,3 +129,190 @@ export function logEvent(message: string): void {
 export function isModelType(value: string): value is ModelType {
   return Object.values<string>(ModelType).includes(value);
 }
+
+export function mapModel2toOldModel(model2: ThreatDragonModel2): ThreatDragonModel {
```

Comment:
When I start the application in a docker setup, download [the V2 example model](https://owasp.org/www-project-threat-dragon/assets/schemas/owasp.threat-dragon.schema.V2.json) and try to start a game, I get an internal server error originating from this function:

```
threats-server  | TypeError: Cannot read properties of undefined (reading 'description')
threats-server  |     at mapModel2toOldModel (/app/packages/shared/dist/utils/utils.js:106:41)
threats-server  |     at /app/apps/server/build/endpoints.js:57:62
threats-server  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
threats-server  |     at async cors (/app/node_modules/@koa/cors/index.js:109:16) TypeError: Cannot read properties of undefined (reading 'description')
threats-server  |     at mapModel2toOldModel (/app/packages/shared/dist/utils/utils.js:106:41)
threats-server  |     at /app/apps/server/build/endpoints.js:57:62
threats-server  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
threats-server  |     at async cors (/app/node_modules/@koa/cors/index.js:109:16)
threats-server  | 
threats-server  |   InternalServerError: Internal Server Error
threats-server  |       at Object.throw (/app/node_modules/koa/lib/context.js:97:11)
threats-server  |       at /app/apps/server/build/endpoints.js:102:18
threats-server  |       at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
threats-server  |       at async cors (/app/node_modules/@koa/cors/index.js:109:16)
```

This is probably a strong indicator that tests are missing for this function.